guides/v2.2/test/testing.md

```diff
@@ -104,16 +104,36 @@ Please see the [System Administrators Guide on Running Tests][] for more informa
 
 Each of the test types listed above corresponds to a subdirectory in `<magento2 root dir>/dev/tests`.
```

Comment:
Consider: "Each test type described above corresponds to a subdirectory in `<magento2 root dir>/dev/tests`, as shown here:"

Reply:
Thanks.